cfme/infrastructure/provider/virtualcenter.py

```diff
@@ -56,7 +56,8 @@ def from_config(cls, prov_config, prov_key, appliance=None):
                    appliance=appliance)
 
     def as_fill_value(self):
-        return {'name': self.name, 'prov_type': 'VMware vCenter'}
+        return {'name': self.name,
+                'prov_type': 'VMware vCenter'}
```

Comment:
It is a pity that I have to get rid of as_fill_value. code looked much better with it.